src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.style.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const StyledDragContainer = styled.div`
+  left: 0;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  pointer-events: none;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+  box-shadow: 0 10px 20px 0 rgba(0,20,29,0.2), 0 20px 40px 0 rgba(0,20,29,0.1);
+`;
+
+const StyledDragWrapper = styled.div``;
```

Comment:
I don't think we _have_ to use `styled.div` here. I guess we could probably use something like this instead:
```jsx
const StyledDragWrapper = (props) => <div>{props.children}</div>;
```
But I think using `styled.div` is cleaner, and also makes it clear that CSS styling can be easily added to the component in the future if needed.

Reply:
I just wonder should we use `StyledComponent` without any styling. I know it is better for the future if we want to add styles... but what if we won't? I think it's like adding `className` to the component without any styling. It can causes a confusion

Reply:
Fair point. It may be a case of [YAGNI](https://en.wikipedia.org/wiki/YAGNI) :slightly_smiling_face:

Refactored in d4f2a93 to remove `StyledDragWrapper` :thumbsup: